tests/src/test/scala/common/Wsk.scala

```diff
@@ -869,7 +869,8 @@ class WskApi()
             { apiname map { a => Seq("--apiname", a) } getOrElse Seq() } ++
             { swagger map { s => Seq("--config-file", s) } getOrElse Seq() } ++
             { responsetype map { t => Seq("--response-type", t) } getOrElse Seq() }
-        cli(wp.overrides ++ params, expectedExitCode, showCmd = true, env=Map("WSK_CONFIG_FILE" -> cliCfgFile.getOrElse("")))
+        val envMap = { cliCfgFile map {f => Map("WSK_CONFIG_FILE" -> f) } getOrElse Map() }
```

Comment:
after thinking more about the map ++ reversal proposal, don't it since it seems more logical for method env parameters should always be overriding the system.env